### ISSUE TITLE
octalLiteral warns when binary literal is used

### ISSUE THREAD

**Issue by beantowel** [go-critic/go-critic#1149]
```go
const (
	bit1   = uint8(0b_00000001)
	minus1 = uint8(0b_11111111)
	nBit   = 3
)
```
This binary literal looks clear enough to distinguish from normal digits, but we got "octalLiteral" warning. go-critic should fix this warning for binary literals.

**Comment by cristaloleg** [comment]
Thanks for reporting, will try to fix today 😉

**Comment by cristaloleg** [comment]
Have tried in the latest go-critic and do not see a problem, can you share what version do you use? how does the error looks like? thanks.

**Comment by quasilyte** [comment]
We need to add some base-2 literals to our negative tests suite for octalLiteral anyway.

**Comment by beantowel** [comment]
I am using `github.com/go-critic/go-critic v0.5.6`, the error looks like: 
```bash
mirror/cron.go:18:11: octalLiteral: suspicious octal args in `uint8(0b_00000001)` (gocritic)
	bit1   = uint8(0b_00000001)
```

**Comment by quasilyte** [comment]
It looks like it's a warning from the old go-critic version.
We're giving a different warning now for octalLiteral.
